test/plugin/scenarios/guava-cache-scenario/support-version.list

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+23.0
```

Comment:
Can this plugin work with the version after 23.0? 
I notice, it is been almost 4 years ago since it was released.

Reply:
It cannot work with the version after 23.0,because some critical classes were changed.

Reply:
As 23 is already a very old version, let's keep limited versions, but true, should not only 23.0

Reply:
Thanks,I will fix it.